src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/SlimefunArmorPiece.java

```diff
@@ -29,4 +36,12 @@ public SlimefunArmorPiece(Category category, SlimefunItemStack item, RecipeType
         return effects;
     }
 
+    /**
+     * This returns the armor set ID of this {@link SlimefunArmorPiece}.
+     *
+     * @return The set ID, <code>null</code> if no set ID is found.
+     */
+    public String getSetID() {
+        return setID;
+    }
```

Comment:
Please use a NamespacedKey for this rather than a String, otherwise you could run into ID conflicts.